arrow-data/src/bitmap.rs

```diff
@@ -68,12 +68,23 @@ impl Bitmap {
         self.bits
     }
 
-    /// Returns the total number of bytes of memory occupied by the buffers owned by this [Bitmap].
+    /// Returns the total number of bytes of memory occupied by the
+    /// buffers owned by this [Bitmap].
+    ///
+    /// Note that this [`Bitmap`] may only refer to a subset of the
+    /// data in the underlying [`Buffer`], but the size includes the
+    /// entire size of the buffer.
```

Comment:
I think this sentence basically tells that this Bitmap is a slice of the Buffer?

Reply:
Yes that was the intent

```suggestion
    /// Note that this [`Bitmap`] may only refer to a subset of the
    /// data in the underlying [`Buffer`] (it is a slice)  but the return value includes the
    /// entire size of the buffer.
```